platform/src/main/java/org/stellar/anchor/platform/service/AnchorMetrics.java

```diff
@@ -1,10 +1,12 @@
 package org.stellar.anchor.platform.service;
 
 public enum AnchorMetrics {
+  SEP6_TRANSACTION_OBSERVED("sep6.transaction.observed"),
```

Comment:
WDYT to merge `AnchorMetrics.java` with `MetricsConstants.java`?

Reply:
I think the metrics need to be refactored quite a bit. If it's all the same to you, I would like to do this in a separate PR.

Reply:
It's ok to be in a separate PR.